sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-file-handler.ts

```diff
@@ -33,9 +34,11 @@ import { ECertFullTimeResponseRecord } from "./e-cert-full-time-integration/e-ce
 import { ProcessSFTPResponseResult } from "../models/esdc-integration.model";
 import { ESDCIntegrationConfig } from "../../types";
 import { ESDCFileResponseDTO } from "../../route-controllers/esdc-integration/models/esdc-model";
+import { DisbursementScheduleWithStopFullTimeBCFundingStatus } from "../../services/disbursement-schedule-service/disbursement-schedule.models";
 
 const ECERT_FULL_TIME_SENT_FILE_SEQUENCE_GROUP = "ECERT_FT_SENT_FILE";
 const ECERT_PART_TIME_SENT_FILE_SEQUENCE_GROUP = "ECERT_PT_SENT_FILE";
+const STRING_ZERO = "0";
```

Comment:
I would recommend removing this const. It does not make the code more or less readable.